### PR TITLE
CAPV: Release v28.0.0.

### PR DESCRIPTION
**Pull request by njuettner**
Towards: https://github.com/giantswarm/roadmap/issues/3716

### Checklist

- [x] Roadmap issue created
- [x] Release uses latest stable Flatcar
- [x] Release uses latest Kubernetes patch version

### Triggering E2E tests

To trigger the E2E test for each new Release added in this PR, add a comment with the following:

`/run releases-test-suites`

If you want to trigger conformance tests, you can do so by adding a comment similar to the following:

`/run conformance-tests PROVIDER=capa RELEASE_VERSION=29.1.0`

For more details see the [README.md](/README.md#running-tests-against-prs).

### CODE REVIEW

**Comment by njuettner** [comment]
/run releases-test-suites TARGET_SUITES=./providers/vsphere/upgrade

**Comment by njuettner** [comment]
/run releases-test-suites TARGET_SUITES=./providers/vsphere/upgrade TARGET_RELEASES=vsphere-28.0.0

**Comment by njuettner** [comment]
/run releases-test-suites TARGET_SUITES=./providers/vsphere/standard

**Review by glitchcrab — APPROVED** [review]

File: vsphere/v28.0.0/release.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  components:
+  - name: cluster-vsphere
+    catalog: cluster
+    version: 0.65.0
+  - name: flatcar
+    version: 3815.2.5
+  - name: kubernetes
+    version: 1.28.12
+  - name: os-tooling
+    version: 1.15.0
```
Comment (Gacko) [inline vsphere/v28.0.0/release.yaml:100]: Can you maybe go for v1.20.1 here? If required, we can also trigger a build for this combination of provider, Flatcar, Kubernetes and tooling.

**Review by Gacko — COMMENTED** [review]
Apart from my comment, I assume we only added the apps of the `cluster-vsphere` chart and kept the versions and dependencies as they are currently defined there, right?

**Comment by Gacko** [comment]
Also maybe consider re-generating the `releases.json` by running `go run renovate.go` upfront. 🙂 This spares the follow-up commit by CI.

**Pull request merged**